src/Sql/dbo/Stored Procedures/OrganizationUser_ReadOccupiedSeatCountByOrganizationId.sql

```diff
@@ -33,5 +33,33 @@ BEGIN
                 -- ACCEPTED status: When SponsoredOrganizationId is not null and ValidUntil is null or in the future
                 (SponsoredOrganizationId IS NOT NULL AND (ValidUntil IS NULL OR ValidUntil > GETUTCDATE()))
             )
-        )
+        ) as Sponsored,
+        (
+            -- Count organization users
+            SELECT COUNT(1)
+            FROM [dbo].[OrganizationUserView]
+            WHERE OrganizationId = @OrganizationId
+            AND Status >= 0 --Invited
+        ) + 
+        (
+            -- Count admin-initiated sponsorships towards the seat count
+            SELECT COUNT(1)
+            FROM [dbo].[OrganizationSponsorship]
+            WHERE SponsoringOrganizationId = @OrganizationId
+            AND IsAdminInitiated = 1
+            AND (
+                -- Not marked for deletion - always count
+                (ToDelete = 0) 
+                OR
+                -- Marked for deletion but has a valid until date in the future (RevokeWhenExpired status)
+                (ToDelete = 1 AND ValidUntil IS NOT NULL AND ValidUntil > GETUTCDATE())
+            )
+            AND (
+                -- SENT status: When SponsoredOrganizationId is null
+                SponsoredOrganizationId IS NULL
+                OR
+                -- ACCEPTED status: When SponsoredOrganizationId is not null and ValidUntil is null or in the future
+                (SponsoredOrganizationId IS NOT NULL AND (ValidUntil IS NULL OR ValidUntil > GETUTCDATE()))
+            )
+        ) as Total
```

Comment:
Do we still need this given that `OrganizationSeatCounts` has its own getter which calculates the total? I think we want that getter OR this calculation in the sproc, not both.

I suggest the getter on the model is the simpler solution.

---

However, if you did want to calculate it in the sproc, this is how to do it without repeating code:

e.g.

```sql
SELECT Subquery.Users, Subquery.Sponsorships, Subquery.Users + Subquery.Sponsorships as Total
FROM (
  SELECT ... as Users,
  ... as Sponsored
) Subquery
```

By putting it in a sub-query you can reference the results in the outer query. [SO reference](https://stackoverflow.com/a/62825452)